independent-publisher-connectors/OpenAI/apiDefinition.swagger.json

```diff
@@ -18,6 +18,462 @@
+                "id": {
+                  "type": "string",
+                  "description": "id"
+                },
+                "object": {
+                  "type": "string",
+                  "description": "object"
+                },
+                "created": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "created"
+                },
+                "model": {
+                  "type": "string",
+                  "description": "Which GPT3 Engine was used",
+                  "title": "Model",
+                  "x-ms-visibility": "internal"
+                },
+                "choices": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "text": {
+                        "type": "string",
+                        "description": "Completion text",
+                        "title": "Text"
+                      },
+                      "index": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "Number of completion",
+                        "title": "Index"
+                      },
+                      "finish_reason": {
+                        "title": "Finish reason",
+                        "type": "string",
+                        "description": "Reason why the text finished (stop condition / natural end / length)"
+                      },
+                      "usage": {
+                        "type": "object",
+                        "properties": {
+                          "prompt_tokens": {
+                            "type": "integer",
+                            "format": "int32",
+                            "description": "Number of tokens in the prompt",
+                            "title": "Prompt Tokens"
+                          },
+                          "completion_tokens": {
+                            "type": "integer",
+                            "format": "int32",
+                            "description": "Number of tokens in the completion",
+                            "title": "Completion Tokens"
+                          },
+                          "total_tokens": {
+                            "type": "integer",
+                            "format": "int32",
+                            "description": "Total number of tokens in prompt and completion",
+                            "title": "Total Tokens"
+                          }
+                        }
+                      }
+                    }
+                  },
+                  "description": "Returned Completion(s)"
+                }
+              }
+            }
+          }
+        },
+        "summary": "GPT3 Completes your prompt",
+        "description": "GPT3 Completes your prompt",
+        "operationId": "Completion_v2",
```

Comment:
@PowerRobin all operationIds should be in Camel Case. please change to "CompletionV2". Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations). please fix all operationIds in the swagger file.

Reply:
Thanks for pointing those out @Amjed-Ayoub 

Made the changes.